### PR TITLE
Add a shortcode for the image compare slider

### PR DESCRIPTION
**Pull request by SIGSTACKFAULT**
Adopting #710.

Rebased onto main and added documentation.

Usage:
```
    {{ compare_slider(
        // mandatory arguments
        left_title="Left Title",
        left_image="left_image.png",
        right_title="Right Title",
        right_image="right_image.png",
        // optional arugments
        start_slider_at="40%", // 0% is completely showing right_image; 100% is completely showing left_image
        left_alt="alt text",   // left image `alt` attribute; defaults to left_title
        right_alt="alt text"   // right image `alt` attribute; defaults to right_title
    ) }}
```

### CODE REVIEW

**Comment by ickk** [comment]
does this still have acceptable fallback behaviour when javascript is disabled in the client?

File: sass/components/_image_compare.scss [hunk 1]
```diff
@@ -1,3 +1,11 @@
+p.image-compare-instruction {
+  font-weight: bold;
+  margin-bottom: 0;
+}
+div.image-compare-container {
+  max-height: 720px;
+  aspect-ratio: 16 / 9;
```
Comment (doup) [inline sass/components/_image_compare.scss:7]: **Nit**: this probably can be removed? The shortcode adds the `aspect-ratio` via style attribute.

```suggestion
```

**Review by doup — APPROVED** [review]
I haven't tested it locally but looks good.

@ickk from what I gather it just adds the shortcode + some extra wrapper (`image-compare-container`) to control the `max-height`. This doesn't touch anything JS related, so it should work same as in prod?

Btw, in `main` it shows the image split in two but the slider can't be moved:
image

**Comment by ickk** [comment]
can you explain what is changing with the 3 pngs?

**Comment by ickk** [comment]
oh I see in your commit message 
> Also replace the aa.png picture with a comparison slider, taking
advantage of the new feature.

**Review by ickk — APPROVED** [review]

**Review by BD103 — APPROVED** [review]
Thanks for taking care of this! 🚀

**Comment by doup** [comment]
> Btw, in `main` it shows the image split in two but the slider can't be moved

Ooops, I meant to show the behavior *when JS is disabled*; with JS enabled the slider can be moved; it's working on prod/`main`.

Another discussion is if the slider should show the images next to each other when JS is disabled. But that's for another ticket.

**Review by cart — APPROVED** [review]

**Comment by SIGSTACKFAULT** [comment]
<3

**Pull request merged**